src/cryptonote_config.h

```diff
@@ -182,6 +182,7 @@
 #define HF_VERSION_EXACT_COINBASE               13
 #define HF_VERSION_CLSAG                        13
 #define HF_VERSION_DETERMINISTIC_UNLOCK_TIME    13
+#define HF_VERSION_VIEW_TAGS                    15
```

Comment:
Isn't `14` the next hardfork? Is this PR being aimed at a summer hardfork? Maybe @selsta can chime in.

Reply:
According to https://github.com/monero-project/monero#scheduled-software-upgrades we currently are on v14

Reply:
(I was looking here: https://github.com/monero-project/monero/blob/a9cb4c082f1a815076674b543d93159a2137540e/src/hardforks/hardforks.cpp)